net-p2p/xmr-stak/files/xmr-stak

```diff
@@ -0,0 +1,28 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms fo the GNU General Public License, v2
+# /etc/init.d/xmr-stak
+name="xmr-stak"
+description="Unified all-in-one Monero miner"
+
+depend() {
+	need net
```

Comment:
Why "need net"? See https://bugs.gentoo.org/439092

Reply:
I wasn't aware of that, thanks! I changed it to use `use dns net`